astropy_sphinx_theme/bootstrap-astropy/layout.html

```diff
@@ -6,6 +6,9 @@
 {# Add the google webfonts needed for the logo #}
 {% block extrahead %}
 <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,600' rel='stylesheet' type='text/css'/>
+{% if astropy_project_menubar == true %}
+<link type='text/css' href='_static/custom.css' />
+{% endif %}
```

Comment:
@kakirastern - is this needed anymore?

Reply:
Yup, it is need, for people to opt in or out of the `astropy_project_menubar` as an option.

Reply:
Sorry, I confused this with something else. It is indeed not needed any more and has been removed.